examples/Training/java/src/training/RawDataAccess.java

```diff
@@ -141,7 +141,9 @@ private void retrieveTile()
             }
         } catch (Exception e) {
             throw new Exception("Cannot read the tiles", e);
-        } 
+        } finally {
+            rdf.close();
```

Comment:
Does this suggest eventually making the `Facility` interfaces subclass `Closeable`?